dev/promote_bom.py

```diff
@@ -91,10 +144,14 @@ def init_argument_parser(cls, parser):
                         help='The changelog to publish during this promotion.')
     parser.add_argument('--github_token', default='', required=True,
                         help="The GitHub user token with scope='gists' to write gists.")
+    parser.add_argument('--github_user', default='', required=True,
```

Comment:
The gist user would be different than the `--github_publisher` when we publish to the upstream Spinnaker repos. I think we wouldn't have access to publish a gist as the `spinnaker` user.